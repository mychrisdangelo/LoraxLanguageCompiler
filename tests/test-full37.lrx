int main()
{
	tree <int>t(2);
	int a;
	a = degree(t) + 100;
	print("should be degree 2 = ", degree(t), "\n");
	print("should be degree 3 = ", degree(1[2, 3[], 4[5, 6, 7]]), "\n");
	// print(degree('a'[])); // still not working
	print("should be degree 1 = ", degree("Hello world\n"), "\n");
	print("should print 102 = ", a, "\n");
}