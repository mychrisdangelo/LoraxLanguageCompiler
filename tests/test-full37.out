should be degree 2 = 2
should be degree 3 = 3
should be degree 1 = 1
should print 102 = 102
